tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.js

```diff
@@ -0,0 +1,155 @@
+
+/*
+Go to the import page
+Download the customers sample file
+Import the customer sample file
+Go to customers page
+Check import success
+ */
+describe('Import customers', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+    await files.deleteFile(renamedSampleFilePath);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to import page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToImportPage', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.advancedParametersLink,
+      dashboardPage.importLink,
+    );
+
+    await importPage.closeSfToolBar(page);
+
+    const pageTitle = await importPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(importPage.pageTitle);
+  });
+
+  it(`should download ${SampleFiles.customers.name} sample file`, async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'downloadFile', baseContext);
+
+    sampleFilePath = await importPage.downloadSampleFile(page, SampleFiles.customers.name);
+
+    const doesFileExist = await files.doesFileExist(sampleFilePath);
+    await expect(doesFileExist, `${SampleFiles.customers.name} sample file was not downloaded`).to.be.true;
+  });
+
+  describe('Import file', async () => {
+    it(`should upload ${SampleFiles.customers.name} sample text file`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'importFile', baseContext);
+
+      await files.renameFile(sampleFilePath, renamedSampleFilePath);
+
+      const uploadSuccessText = await importPage.uploadSampleFile(
+        page,
+        SampleFiles.customers.dropdownValue,
+        renamedSampleFilePath,
+      );
+      await expect(uploadSuccessText).contain(SampleFiles.customers.name);
+    });
+
+    it('should go to next import file step', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'nextStep', baseContext);
+
+      const panelTitle = await importPage.goToImportNextStep(page);
+      await expect(panelTitle).contain(importPage.importPanelTitle);
+    });
+
+    it('should start import file', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'confirmImport', baseContext);
+
+      const modalTitle = await importPage.startFileImport(page);
+      await expect(modalTitle).contain(importPage.importModalTitle);
+    });
+
+    it('should close import progress modal', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'closeImportModal', baseContext);
+
+      const fileTypeDropdownSelector = await importPage.closeImportModal(page);
```

Comment:
That's a weird `const` name. Is it right ?

Reply:
indeed a little lack of inspiration for this const. I will try to rename it to something more explicit

Reply:
The thing is, when I read the code, I don't understand why it's called `fileTypeDropdownSelector` because I have no idea what is this dropdown your're talking about. If you're checking if the modal was indeed closed, it should be named after the intended action, so something like `isModalClosed` would make more sense IMO.
What do you think ?

Reply:
Yes totally agree with you and @Progi1984 the const name was not accurate and can cause some confusion !
The change is done !